### PR TITLE
#1853 multi-exam-import bug

### PR DESCRIPTION
**Pull request by jcomedouteau**
- Correct logging in case of error with specific pacs error
- Do not delete directly the temp folder when importing

How to test:
- Ask me for data
- Test multi exam import with the data
- -> If error, the import does not stop.
- -> If no error, no error.

### CODE REVIEW

**Comment by michaelkain** [comment]
Hi @jcomedouteau, thank you for your PR. I hope you are doing fine. I am wondering about something and maybe I am wrong. We create an ImportJob par exam, that has a specific folder. Why, when using that folder, each importJob can not clean his workFolder at the end of each import per exam? Why do we require a flag is multipleExam in an ImportJob, that is by default made for one exam only? Maybe I am wrong and with kind regards, Michael

**Comment by michaelkain** [comment]
To be more precise: I mean setting importJob.WorkFolder to the current folder of the exam.

File: shanoir-ng-datasets/src/main/java/org/shanoir/ng/dicom/DIMSEService.java [hunk 1]
```diff
@@ -91,13 +91,14 @@ public void sendDicomFilesToPacs(File directoryWithDicomFiles) throws Exception
 	private void execute(List<String> args) throws Exception {
 		LOG.debug("Calling command: " + args.toString());
 		ProcessBuilder processBuilder = new ProcessBuilder();
+		processBuilder.redirectErrorStream(true);
```
Comment (michaelkain) [inline shanoir-ng-datasets/src/main/java/org/shanoir/ng/dicom/DIMSEService.java:94]: Great!

**Comment by jcomedouteau** [comment]
Hi @michaelkain, welcome back !
You're right, my boolean was a false lead as I thought we had a problem when deleting import folder too soon, but this is not the case. The problem was directly linked to the send to the pacs method execution that was stopping abruptly.
I'm removing this !

**Comment by jcomedouteau** [comment]
@michaelkain as discussed, I added an new log file for DIMSE
image

**Comment by michaelkain** [comment]
Hi, I tested and it works. We need to discuss with Anthony, if 500KB per import are an issue. Means 50MB logs per day for OFSEP, what could be a lot. And please fix the pseudo issue on this PR, before merge. With kind regards, Michael

**Review by michaelkain — APPROVED** [review]
Thank you very much! It works on my machine.

**Pull request merged**